bin/envconfig

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
+
+# setup lsstsw environment
```

Comment:
Are people required to use this script? Or can they use `conda env list` themselves and `conda activate`?

Reply:
Other environment variables are defined here, not only the conda environment to use.
The user may have already defined those in the .bash_profile.
However, if the scope of this tool is to provide a ready to use environment, I think the user is required to use this script.